conans/test/functional/configuration/profile_test.py

```diff
@@ -148,11 +152,19 @@ def install_profile_env_test(self):
         files["conanfile.py"] = conanfile_scope_env
 
         create_profile(self.client.cache.profiles_path, "envs", settings={},
-                       env=[("A_VAR", "A_VALUE")], package_env={"Hello0": [("OTHER_VAR", "2")]})
+                       env=[("A_VAR", "A_VALUE"),
+                            ("PREPEND_VAR", ["new_path", "other_path"])],
+                       package_env={"Hello0": [("OTHER_VAR", "2")]})
 
         self.client.save(files)
         self.client.run("export . lasote/stable")
         self.client.run("install Hello0/0.1@lasote/stable --build missing -pr envs")
+        if platform.system() == "Windows":
+            self._assert_env_variable_printed("PREPEND_VAR", "new_path;other_path")
+        else:
+            self._assert_env_variable_printed("PREPEND_VAR", "new_path:other_path")
+        self.assertNotIn("PREPEND_VAR=new_path;other_path;new_path;other_path", self.client.out)
```

Comment:
Do not check this way. For example, when using Linux, it will never happen, because ``os.pathsep``.
The correct way to robustly check this is something like:

``self.assertEqual(1, str(self.client.out).count("new_path")``